ethereum-common/src/keccak_hasher.rs

```diff
@@ -0,0 +1,24 @@
+use ethereum_types::H256;
+use hash256_std_hasher::Hash256StdHasher;
+use hash_db::Hasher;
+use tiny_keccak::{Hasher as _, Keccak};
+
+/// Concrete `Hasher` implementation for the Keccak-256 hash
+#[derive(Default, Debug, Clone, PartialEq)]
+pub struct KeccakHasher;
+
+impl Hasher for KeccakHasher {
```

Comment:
Isn't it already implemented somewhere?

Reply:
frankly speaking I didn't find it anywhere

Reply:
However there is one but it's not a library - https://github.com/openethereum/parity-ethereum/blob/v2.7.2-stable/util/keccak-hasher/src/